static/js/components/TopSources.jsx

```diff
@@ -32,10 +32,13 @@ const TopSources = () => {
       <p>
         Displaying most-viewed sources
       </p>
-      <ul>
+      <ul style={{ listStyleType: "none" }}>
         {
-          sourceViews.map(({ obj_id, views }) => (
+          sourceViews.map(({ obj_id, views, public_url }) => (
             <li key={`topSources_${obj_id}_${views}`}>
+              <span>
+                <img src={public_url} alt={obj_id} width="50%" height="50%" />
```

Comment:
Make the image clickable to the source and put the styles in `TopSources.css`:
```suggestion
            <Link to={`/source/${obj_id}`}>
                <img className={styles.stamp} src={public_url} alt={obj_id} />
             </Link>
```
Here, the file `TopSources.css` can be something like:
```
.stamp {
    transition: transform 0.1s;
    width: 20%;
    height: 20%;
    margin: 0 auto;
}

.stamp:hover {
  transform: scale(2.0);
}
```
And `styles` can be added in `TopSources.jsx` like (around line 7):
```
import styles from "./TopSources.css";
```
This will look like:
![Aug-04-2020 22-51-37](https://user-images.githubusercontent.com/480799/89376970-b900b280-d6a5-11ea-9797-47f03ababd17.gif)

Reply:
This is really neat! Implemented.